tasks/eva_3004/eva3004/src/uk/ac/ebi/eva/eva3004/qc_eva2959_prod.groovy

```diff
@@ -47,29 +53,31 @@ allRSToCheck.each{assembly, allRSIDs -> allRSIDs.collate(1000).each{rsIDs ->
         prodEnv.mongoTemplate.find(query(where("seq").is(assembly).and("rs").in(rsIDs)), it)}.flatten().findAll{
         it.isAllelesMatch() && Objects.isNull(it.mapWeight)}
     def rsGroupedByHashAndAccessionFromSVE =
-            sves.collectEntries{sve ->
-                def cve = toClusteredVariantEntity(sve)
-                return ["${cve.hashedMessage}_${cve.accession}", sve]
+            sves.groupBy{sve ->
+                def cve = EVADatabaseEnvironment.toClusteredVariantEntity(sve)
+                return ""+ "${cve.hashedMessage}_${cve.accession}"
```

Comment:
Fix a bug to use groupBy instead of collectEntries. The former can support multiple values per key but the latter can't.